homework/config-files/hackyourtemperature/server.js

```diff
@@ -0,0 +1,19 @@
+import express from "express";
+
+const app = express();
+app.use(express.json());
+app.use(express.urlencoded({ extended: true }));
+app.use(express.static("public"));
+app.set("views", "./views");
+app.set("view engine", "ejs");
+
+app.get("", (req, res) => {
```

Comment:
The standard is usually to include a slash here. Just a small improvement 😄 

```suggestion
app.get("/", (req, res) => {
```

Reply:
thank you so much,  I'm gonna correct it